pkg/apis/kubermatic/v1/configuration.go

```diff
@@ -270,6 +272,18 @@ type MachineControllerConfiguration struct {
 	ImageTag string `json:"imageTag,omitempty"`
 }
 
+// OperatingSystemManager configures the image repo and the tag version for osm deployment.
+type OperatingSystemManager struct {
+	// ImageRepository is used to override the OperatingSystemManager image repository.
+	// It is recommended to use this field only for development, tests and PoC purposes.For production environments.
+	// it is not recommended, to use this field due to compatibility with the overall KKP stack.
+	ImageRepository string `json:"imageRepository,omitempty"`
```

Comment:
Everywhere else we call these fields "DockerRepository". Can we do the same here, same for "DockerTag" instead of "ImageTag"?

Reply:
Honestly image repo/tag is the correct term here, not `DockerRepository`

Reply:
I would also rather start using ImageTag and ImageRepository over Docker tag/repo. It makes more sense even if it costs us some consistency.